docs/content/preview/explore/multi-region-deployments/synchronous-replication-ysql.md

```diff
@@ -36,134 +36,134 @@ type: docs
 
```

Comment:
ysql and ycql pages appear identical? do we need to split them?

Reply:
They are very similar, but not identical (e.g. "open `ycqlsh` by running the following commands" and then examples that use ycqlsh).

Reply:
Asynchronous example, yes, but synchronous page never uses a shell only yb-admin and yb-ctl

Reply:
Marco has just gotten back to me on the "Add tables" section in _ycql. I'll create a new ticket for that.